index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name=”viewport” content=”width=device-width, initial-scale=1″ charset="UTF-8">
+    <link rel="stylesheet" href="css/style.css">
+    <link href='http://fonts.googleapis.com/css?family=Lato:300,400,500,600,700' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Paytone One' rel='stylesheet'>
+    <title>Conference Page</title>
+  </head>
+  <body onload="initialExec();">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
+    <script src="js/index.js"></script>
+    <div id="container">
+
+      <!-- header -->
+      <header id="header">
+        <div id="nav-banner">
+          <ul id="nav-banner-list">
+            <li id="nav-banner-list-item"><img id="icon-face" src="images/face.png"></li>
+            <li id="nav-banner-list-item"><img id="icon-twitter" src="images/twitter.svg"></li>
+            <li id="nav-banner-list-item">English</li>
+            <li id="nav-banner-list-item">My Page</li>
+            <li id="nav-banner-list-item">Logout</li>
+          </ul>
+        </div>
+        <div id="navigation">
+          <a id="navigation-logo" href="index.html"></a>
+          <ul id="navigation-list">
+            <li id="nav-navigation-list-item"><a href="index.html">Home</a></li>
+            <li id="nav-navigation-list-item"><a href="about.html">About</a></li>
+            <li id="nav-navigation-list-item"><a href="index.html#program">Program</a></li>
+            <li id="nav-navigation-list-item"><a href="index.html#speakers-section">Join</a></li>
+            <li id="nav-navigation-list-item"><a href="index.html#partners">Sponsor</a></li>
+            <li id="nav-navigation-list-item"><a href="about.html#see-past">News</a></li>
```

Comment:
[ OPTIONAL ] What do you think if you add different ids on each section, and use them each navigation option, so once clicked from the navbar it leads the user to the correct section